### PR TITLE
Template advancements

### PR DESCRIPTION
**Pull request by benja**
- Made template mobile responsive
- Added support for nested sub-navigations
- Added new footer component

### CODE REVIEW

File: packages/template/index.html [hunk 1]
```diff
@@ -69,6 +225,62 @@ <h1 class="text-white">
...
+      <div slot="content">El Contento</div>
+      <div slot="content">El Contento</div>
+      <div slot="content">El Contento</div>
+      <div slot="content">El Contento</div>
+      <div slot="content">El Contento</div>
+      <div slot="content">El Contento</div>
+      <div slot="content">El Contento</div>
+      <div slot="content">El Contento</div>
+      <div slot="content">El Contento</div>
+      <div slot="content">El Contento</div>
```
Comment (digitalsadhu) [inline packages/template/index.html:270]: whats the reason behind repeating this div?

File: packages/template/index.html [hunk 2]
```diff
@@ -49,17 +49,173 @@
...
+                    }
+                  ]
+                },
+                {
+                  "title": "Forms",
+                  "open": false,
+                  "items": [
+                    {
+                      "title": "Checkbox",
+                      "href": "?page=checkbox"
```
Comment (digitalsadhu) [inline packages/template/index.html:93]: How do the mix of query params like this line and hrefs that are relative urls work? eg. `"href": "/input"`
Are the query params overrides?

**Review by digitalsadhu — APPROVED** [review]
Looks good

File: packages/template/index.html (hunk 1 above)
Comment (benja) [reply packages/template/index.html:270]: Just for testing scrolling (just duplicated quickly while coding), can be removed. Could've probably put a bunch of H1s in the div itself.

File: packages/template/index.html (hunk 2 above)
Comment (benja) [reply packages/template/index.html:93]: No, it is just for testing paths and active state locally. It checks the URL whether there's a match and if there is it sets the active element in the sidebar.

**Pull request merged**